GlucoseOnFhir/GlucoseMeterViewController.swift

```diff
@@ -251,21 +355,43 @@ class GlucoseMeterViewController: UITableViewController, GlucoseProtocol, FHIRPr
         return nil
     }
 
-    
-    //MARK
-    func searchForPatient() {
-            let searchDict:[String:Any] = [
-            "given":"Lisa",
-            "family":"Simpson"
+    func searchForPatient(given:String, family:String, callback: @escaping FHIRSearchBundleErrorCallback) {
+        print("GlucoseMeterViewController: searchForPatient")
+        let searchDict:[String:Any] = [
+            "given":given,
+            "family":family
         ]
         
-        fhir.searchForPatient(searchParameters:searchDict)
+        logEvent(event: "searching for patient \(given) \(family)")
+        
+        FHIR.sharedInstance.searchForPatient(searchParameters: searchDict) { (bundle, error) -> Void in
+            guard error == nil else {
+                print("error searching for patient: \(error)")
+                callback(bundle, error)
+                return
+            }
```

Comment:
again, i would try to simplify the return by only having a single call to `callback()`